php/example_code/dynamodb/Readme.md

```diff
@@ -0,0 +1,84 @@
+# Amazon DynamoDB code examples for the AWS SDK for PHP v3
+
+## Purpose
+
+Shows how to use the AWS SDK for PHP v3 to get started using operations in Amazon DynamoDB. Learn to create tables, add
+items, update data, create custom queries, and delete tables and items.
+
+*Amazon DynamoDB is a fully managed NoSQL database service that provides fast and predictable performance with seamless
+scalability.*
+
+## ⚠️ Important
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. 
+At most, grant only the minimum permissions required to perform the task. 
+For more information, see 
+[Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. 
+For more information, see 
+[AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single actions
+
+**Actions**
+
+* [Creating a table](dynamodb_basics/GettingStartedWithDynamoDB.php)(`create_table`)
+* [Deleting a table](DynamoDBService.php)(`delete_table`)
+* [Deleting an item from a table](dynamodb_basics/GettingStartedWithDynamoDB.php)(`delete_item`)
+* [Getting an item from a table](dynamodb_basics/GettingStartedWithDynamoDB.php)(`get_item`)
+* [Putting an item into a table](dynamodb_basics/GettingStartedWithDynamoDB.php)(`put_item`)
+* [Putting items loaded from a JSON file into a table](dynamodb_basics/GettingStartedWithDynamoDB.php)
+  (`put_item`)
+* [Querying items by using a key condition expression](dynamodb_basics/GettingStartedWithDynamoDB.php)
+  (`query`)
+* [Scanning a table for items](dynamodb_basics/GettingStartedWithDynamoDB.php)
+  (`scan`)
+* [Updating an item in a table](dynamodb_basics/GettingStartedWithDynamoDB.php)
+  (`update_item`)
+
+### Scenario examples
+
+* [Getting started with DynamoDB](dynamodb_basics/GettingStartedWithDynamoDB.php)
```

Comment:
Same note as before, about using the [template wiki](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template) format if it makes sense. 

For example: 

### Scenarios

Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.

Reply:
Fixed.